lib/elastic_record/relation/hits.rb

```diff
@@ -13,7 +13,7 @@ def search_hits
 
       def search_results
         @search_results ||= begin
-          options = { typed_keys: true }
+          options = { typed_keys: true, track_total_hits: true }
```

Comment:
@matthuhiggins this causes the optimization you pointed me to to not be hit for nearly everything we do.

https://www.elastic.co/blog/faster-retrieval-of-top-hits-in-elasticsearch-with-block-max-wand

Matching does not hit this, however, because we do _msearch which needs to be parsed differently.